x/concentrated-liquidity/fees.go

```diff
@@ -253,6 +253,51 @@ func (k Keeper) collectFees(ctx sdk.Context, poolId uint64, owner sdk.AccAddress
 	return feesClaimed, nil
 }
 
+// queryClaimableFees queries the fee accumulator for the position given by pool id, owner, lower tick and upper tick.
+// It returns the outstanding fees that can be claimed by the owner.
+// Returns error if:
+// - pool with the given id does not exist
+// - position given by pool id, owner, lower tick and upper tick does not exist
+// - other internal database or math errors.
+func (k Keeper) queryClaimableFees(ctx sdk.Context, poolId uint64, owner sdk.AccAddress, lowerTick int64, upperTick int64) (sdk.Coins, error) {
+	cacheCtx, _ := ctx.CacheContext()
+	feeAccumulator, err := k.getFeeAccumulator(cacheCtx, poolId)
+	if err != nil {
+		return nil, err
+	}
+
+	positionKey := formatFeePositionAccumulatorKey(poolId, owner, lowerTick, upperTick)
+
+	hasPosition, err := feeAccumulator.HasPosition(positionKey)
+	if err != nil {
+		return nil, err
+	}
+
+	if !hasPosition {
```

Comment:
This is an open ended question, but Im not sure if we should error here. 
For the bank balance query for example, it does not error upon querying for an account that does not exist in state, it returns empty response. But again, this is an open-ended question, please feel free to resolve this if you otherwise disagree

Reply:
I vote for lets flag this in an issue, and start a discussion in our shared-SDK team slack channel for whats the standard we should have for clients.

Reply:
Issue created and asked in the SDK channel https://github.com/osmosis-labs/osmosis/issues/4521